lib/Serge/Engine/Plugin/parse_yaml.pm

```diff
@@ -134,12 +141,30 @@ sub parse {
         die $error_text;
     }
 
+    if ($self->{data}->{yaml_mode} eq 'rails') {
```

Comment:
Instead of comparing strings like that in two places, I'd suggest setting the boolean flag and using that. See https://github.com/evernote/serge/blob/master/lib/Serge/Engine/Plugin/parse_xml.pm#L54 and https://github.com/evernote/serge/blob/master/lib/Serge/Engine/Plugin/parse_xml.pm#L409 as an example.